src/css/page-section.scss

```diff
@@ -6,6 +6,10 @@
   transition: max-height 0.25s ease-in;
   max-height: none;
 
+  &:first-child {
+    margin-top: 48px;
+  }
```

Comment:
One thing has missed for the mobile device. It need to add breakpoint for mobile device.

Reply:
Please check out my latest commit to this branch. It has the breakpoint for mobile devices.